tests/ui/imports/no-reexports-but-used.rs

```diff
@@ -0,0 +1,14 @@
+// run-pass
```

Comment:
It's a bit confusing that only `﻿run-pass` triggers a warning message, while `﻿build-pass` and `﻿check-pass` do not.  It appears that the lint level is set to ﻿`allow` under ﻿`build-`pass or `﻿check-pass`.

And the changes observed in other test cases are due to this reason.

I'm not sure if this is the expected behavior, so I'm leaving this comment.

Reply:
I discovered the issue was due to the usage of [AllowUnused::Yes](https://github.com/rust-lang/rust/blob/master/src/tools/compiletest/src/runtest.rs#L1899). As a resolution, I've incorporated the `﻿#[warn(unused_imports)]` attribute.